PuppetforgeModulePdkVersion Service Test Failing

:clock11: **When did the problem start?** 
7th Jan 2020 / [daily test run 838 ](https://circleci.com/gh/badges/daily-tests/838)

<!-- Indicate when the problem started -->

:camera: **Live badge**

<!-- Provide a link to the live badge in plain text and markdown.  -->

https://img.shields.io/puppetforge/pdk-version/tragiccode/azure_key_vault
![](https://img.shields.io/puppetforge/pdk-version/tragiccode/azure_key_vault)

:wrench: **Is the live badge working?**
Yes, seems to be
<!-- Indicate whether or not the live badge is working. -->

:link: **CircleCI link**
https://circleci.com/gh/badges/daily-tests/848

<!-- Provide a link to the failing test in CircleCI. -->

:beetle: **Stack trace**

```
1. PuppetforgeModulePdkVersion [live] PDK version (library doesn't use the PDK) [ GET /camptocamp/openssl.json ]
  AssertionError: message mismatch: expected 'v1.14.1' to equal 'none'
```

:bulb: **Possible solution**
Looks like the most likely problem is with the target being used for that particular test, so it could just need a different target

<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

<!-- Love Shields? Please consider donating $10 to sustain our activities:
👉  https://opencollective.com/shields -->

--- a/services/puppetforge/puppetforge-module-pdk-version.tester.js
+++ b/services/puppetforge/puppetforge-module-pdk-version.tester.js
@@ -11,7 +11,7 @@ t.create('PDK version')
   })
 
 t.create("PDK version (library doesn't use the PDK)")
-  .get('/camptocamp/openssl.json')
+  .get('/puppet/yum.json')
   .expectBadge({
     label: 'pdk version',
     message: 'none',

